### PR TITLE
internal/website: use code font for reference page titles in sidebar

### PR DESCRIPTION
**Pull request by zombiezen**
Split MySQL and PostgreSQL packages out to match. Removed list on front page that would have been broken by this change.

This supersedes #2423

### CODE REVIEW

File: internal/website/content/ref/mysql.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "mysql"
+date: 2019-03-11T10:02:56-07:00
+aliases:
+- /ref/sql/
+- /pages/sql/
+---
+
+The `mysql` packages provide functions for connecting to [MySQL][] databases
```
Comment (eliben) [inline internal/website/content/ref/mysql.md]: package provides

File: internal/website/content/ref/mysql.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+for both on-premise and cloud-provided instances using the [`*sql.DB`][] type
+from the Go standard library. Database connections opened using these
```
Comment (eliben) [inline internal/website/content/ref/mysql.md]: this package

File: internal/website/content/ref/postgres.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: "postgres"
+date: 2019-06-27T13:03:45-07:00
+---
+
+The `postgres` package provides functions for connecting to [PostgreSQL][]
+databases for both on-premise and cloud-provided instances using the
+[`*sql.DB`][] type from the Go standard library. Database connections opened
+using these packages will automatically collect diagnostic information via
```
Comment (eliben) [inline internal/website/content/ref/postgres.md]: this package

Comment (zombiezen) [reply internal/website/content/ref/postgres.md]: Done.

File: internal/website/content/ref/mysql.md (hunk 1 above)
Comment (zombiezen) [reply internal/website/content/ref/mysql.md]: Done.

File: internal/website/content/ref/mysql.md (hunk 2 above)
Comment (zombiezen) [reply internal/website/content/ref/mysql.md]: Done.

**Pull request merged**